pacu/modules/sns__subscribe/main.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+import argparse
+import re
+from copy import deepcopy
+
+from pacu.core.lib import strip_lines
+from pacu import Main
+
+module_info = {
+    "name": "sns__subscribe",
+    "author": "h00die & 6a6f656c of nDepth Security",
+    "category": "LATERAL_MOVE",  # or maybe persistence? kind of depends what may come over the topic, like creds
+    "one_liner": "Subscribe to a Simple Notification Service (SNS) topic",
+    "description": strip_lines(
+        """
+        This module will attempt to subscribe to a topic (arn) via an email address.
+        """
+    ),
+    "services": ["SNS"],
+    "prerequisite_modules": [],
+    "arguments_to_autocomplete": [],
+}
+
+parser = argparse.ArgumentParser(add_help=False, description=module_info["description"])
+
+parser.add_argument(
+    "--topic",
```

Comment:
This could be a `--topics` and take multiple ARNs to allow subscribing to many at once?

This would require a loop when subscribing to them below obviously, but would work well along with the `--all` flag suggested below.